src/css/application.css

```diff
@@ -329,14 +329,32 @@ body {
   flex: 1 1 auto;
   display: flex;
   flex-direction: row;
+  height: 100vh;
```

Comment:
If `height: 100vh` is placed in `environment-column` instead, when all containers are closed, the site collapses onto the `POPCODE` sidebar and doesn't look that great.